src/makeCalculator.js

```diff
@@ -36,8 +36,35 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calc = {
+    result: 0,
+    add(x) {
+      this.result += x;
```

Comment:
```suggestion
      calc.result += x;
```
you can use calc to avoid `.call` and this manipulations